core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -413,7 +412,7 @@ public long handleWayTags(ReaderWay way, long allowed, long relationFlags) {
             flags = accessEncoder.setValue(flags, getAccessValue(way));
 
 
-            if(spatialRuleLookupEnabled()){
+            if (spatialRuleLookupEnabled()) {
```

Comment:
I've seen this in some other PR but forgot to comment: if we make this method public we have to follow java coding rules: `isSpatialRuleLookupEnabled`